Create Glossary stubs S (#3897)
ppy/osu-wiki: wiki/Spectator_aura/en.md

---
stub: true
---

<!-- didn't do anything to this one, but personally don't think we need this article -->

> # Spectator Aura

Reviewer: ```suggestion
# Spectator aura
```

> A mysterious phenomenon that could cause players to [choke](<!-- link -->) their [combo](<!-- link -->), or fail a beatmap from having a spectator(s) spectate their gameplay.

Reviewer: ```suggestion
The **spectator aura** is a mysterious phenomenon which can cause players to [choke](<!-- link -->) their scores or fail a beatmap from having a spectator(s) watch their gameplay.
```
